Makefile

```diff
@@ -145,12 +145,17 @@ dunestrap: $(DUNE_FILES)
 states: dunestrap
 	dune build $(DUNEOPT) dev/shim/coqtop
 
-NONDOC_INSTALL_TARGETS:=coq-core.install coq-stdlib.install coqide-server.install coqide.install coq.install
+LABLGTK:=lablgtk3-sourceview3
+HAS_LABLGTK:=$(shell ocamlfind query -qe $(LABLGTK))
```

Comment:
Would people mind if we were to just remove `coqide` from `NONDOC_INSTALL_TARGETS` and add a dedicated rule for it?
```make
coqide:
        dune build $(DUNEOPT) coqide.install
```
(As a pure CoqIDE user, I certainly would not mind.)